src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -46,7 +46,7 @@ public PanelControl(UI ui, Preferences prefs) {
         ui.registerEvent((PanelClickedEventHandler) e ->
                 setCurrentlySelectedPanel(Optional.of(e.panelIndex)));
         ui.registerEvent((ShowRenamePanelEventHandler) e ->
-                ((FilterPanel) getPanel(e.panelId)).showRenameTextField());
+                ((FilterPanel) getPanel(e.panelId)).initRenameHandler());
```

Comment:
Maybe the method could be named in a way that doesn't mention implementation details, like `startRename` or something?